products/compute/terraform.yaml

```diff
@@ -227,6 +227,11 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
       protocol: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
+        description: |
+          The protocol this BackendService uses to communicate with backends.
+          Possible values are HTTP, HTTPS, HTTP2, TCP, and SSL. The default is
+          HTTP. **NOTE**: HTTP2 is only valid for beta HTTP/2 load balancer
```

Comment:
Since these values will be surfaced in Ansible too, should we make the beta warning generic and surface it in `api.yaml`? `**NOTE**: HTTP2 is a beta value` or similar.

Reply:
sure!